extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -265,6 +265,7 @@ class AmpYoutube extends AMP.BaseElement {
   layoutCallback() {
     // See https://developers.google.com/youtube/iframe_api_reference
     const iframe = createFrameFor(this, this.getVideoIframeSrc_());
+    iframe.setAttribute('title', 'AMP YouTube video');
```

Comment:
Is this right? Should it not be `this.element.title || 'AMP YouTube video'`?

Also, does it need to say "AMP"? I think it could just be "YouTube video".

Basically, the same thing that was done for `amp-viqeo`.

Reply:
Thanks for catching this @westonruter, @realPrimoh looks like we missed this one, could you follow up with the fix?

Reply:
Yup! @westonruter Thanks for catching this!